### DISCUSSION TITLE
reflex run fail（Unable to download package）

### DISCUSSION THREAD

**Discussion by sonic2825** [reflex-dev/reflex#2286] [Q&A]
(base) root@tdserver:/script/python/reflex_test_1# reflex run
────────────────────────────────────────────────────────────────────────────────────────────── Starting Reflex App ──────────────────────────────────────────────────────────────────────────────────────────────
Compiling: ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 9/9 0:00:00
Installing base frontend packages failed with exit code 1
bun add v1.0.13 (f5bf67bd)
 Resolving dependencies

error: ConnectionRefused downloading package manifest axios

error: ConnectionRefused downloading package manifest autoprefixer

error: ConnectionRefused downloading package manifest next-sitemap

error: ConnectionRefused downloading package manifest react-dom

error: ConnectionRefused downloading package manifest next

error: ConnectionRefused downloading package manifest json5

error: ConnectionRefused downloading package manifest universal-cookie

error: ConnectionRefused downloading package manifest socket.io-client

error: ConnectionRefused downloading package manifest postcss

error: ConnectionRefused downloading package manifest silly

error: ConnectionRefused downloading package manifest next-themes

error: ConnectionRefused downloading package manifest react
 Resolved, downloaded and extracted [72]
error: autoprefixer@10.4.14 failed to resolve
error: postcss@8.4.31 failed to resolve
error: axios@1.4.0 failed to resolve
error: json5@2.2.3 failed to resolve
error: next@14.0.1 failed to resolve
error: next-sitemap@4.1.8 failed to resolve
error: next-themes@0.2.0 failed to resolve
error: react@18.2.0 failed to resolve
error: react-dom@18.2.0 failed to resolve
error: silly@ failed to resolve
error: socket.io-client@4.6.1 failed to resolve
error: universal-cookie@4.0.4 failed to resolve
Run with --loglevel debug  for the full log.

**Comment by ElijahAhianyo** [comment]
Might be an issue with your internet connection. Does this happen on every retry?

**Reply by sonic2825** [reply]
Yes, I am in China. Does it have anything to do with the region? Is it possible that the download address is being blocked?
